_posts/2016-12-06-future.md

```diff
@@ -0,0 +1,444 @@
+
+Relevant Issues:
+
+- [Reorganizing the docs layout](https://github.com/babel/babel.github.io/issues/930)
+- [Showcase community plugins](https://github.com/babel/babel.github.io/issues/831)
+- [Adding a resources page](https://github.com/babel/babel.github.io/issues/27)
+- [Learn ES2015: teach others about ES2015+](https://github.com/babel/babel.github.io/issues/994)
+- [REPL: adding example dropdowns](https://github.com/babel/babel.github.io/issues/59)
+- [REPL: supporting all options](https://github.com/babel/babel.github.io/issues/92)
+- [REPL: allow the REPL to import any community plugin from npm](https://github.com/babel/babel.github.io/issues/858)
+- [REPL: Use ASTexplorer](https://github.com/fkling/astexplorer/issues/70)
+- [ASTexplorer: Real Time collaboration](https://github.com/fkling/astexplorer/issues/166)
+
+## Future
+
+These are all things that can be changed or dropped. Some are already in the works and others need a proper discussion/champion.
+
+### [Plugin API Changes](https://github.com/babel/notes/blob/master/2016-08/august-01.md#potential-api-changes-for-traversal)
+
+There is a lot of confusion around how plugins/presets interact regarding ordering. This results in bugs and issues with the config which require users to place plugins before/after others in a non-intuitive way.
+
+This is a fundamental change to the core of Babel which will take some work to figure out.
+
+### Versioning
+
+Since Babel 6 we've used a ["fixed" mode of versioning](https://github.com/lerna/lerna#how-it-works) via Lerna. This is what allows us to release multiple packages at the same time all under the same version (if that package changes). This is nice because you don't have to manually set a version for each package but everything moves together. The only time this is a pain is when one on the packages makes a breaking change: then every package also will bump major versions.
+
+This is explained more in [babel/notes](https://github.com/babel/notes/blob/master/2016-07/july-31.md#future-of-babels-release-process-and-its-ecosystem) but we still need to figure out the best plan of action for the project.
+
+What happens when we need to update a Stage 0 spec to Stage 1 and it's a breaking change to the parser? Are we just going to bump the major version, wait a batch some changes up, or figure out how to do it via multiple versions of plugins?
+
+[Discussion Issue](https://github.com/babel/babel/issues/4950)
+
+#### Changing the Mindset Around Stage X Presets
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">My rule of thumb on how I decide what future features to transpile:<br>&quot;Could I reasonably codemod this if it changes?&quot;<br>Don&#39;t do it otherwise.</p>&mdash; Kent C. Dodds (@kentcdodds) <a href="https://twitter.com/kentcdodds/status/803815749416456196">November 30, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
Stick this script in the layout wherever all other scripts are loaded, rather than in the middle of the post like this?